serveradmin/serverdb/models.py

```diff
@@ -100,8 +100,9 @@ def is_ip_address(ip_interface: Union[IPv4Interface, IPv6Interface]) -> None:
 
 def is_unique_ip(
     ip_interface: Union[IPv4Interface, IPv6Interface],
-    object_id: int,
-    attribute_id: str = None,
+    server: "Server",
```

Comment:
Shouldn't this be Server without double quotes ?

Reply:
The class is defined in the same but later. We need a forward reference.

Reply:
Ah true.